frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.theme.default.css

```diff
@@ -61,9 +61,9 @@ table.treetable tr.expanded span.indenter a {
 }
 
-table.treetable tr.branch {
-  background-color: #f9f9f9;
-}
+/*table.treetable tr.branch {*/
```

Comment:
I've created our own copy of the default stylesheet and stripped it of unnecessary styling instead of loading it and then overwriting